src/components/Button/Button.mdx

```diff
@@ -11,7 +11,7 @@ Buttons are styled semantic buttons that will handle all of the appropriate butt
 
 ## How to Use
 
-There are 3 variants of buttons: primary, secondary, and tertiary. By default, the button will display as primary.
+There are 4 variants of buttons: primary, secondary, alert, and tertiary. By default, the button will display as primary.
```

Comment:
It looks like there are more than 4... there are 8 and we just haven't been good about keeping this updated

Reply:
Right.. Fixed.